processors/project-reference-labeler.ts

```diff
@@ -16,38 +18,111 @@ export function getDefaultConfiguration(): any {
   return <any>{
     'process-with-label': 'feature',
     'column-label-prefix': '> ',
-    'linked-label-prefix': '>> '
+    'linked-label-prefix': '>> ',
```

Comment:
space is important here since we do a startsWith later.  should probably add a check since it's easy to mis and could lead to weird behaviors